[2024-09-01T07:08:39.630+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-01T07:08:39.773+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrap_mobil123.modify_data scheduled__2024-08-31T00:00:00+00:00 [queued]>
[2024-09-01T07:08:39.791+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrap_mobil123.modify_data scheduled__2024-08-31T00:00:00+00:00 [queued]>
[2024-09-01T07:08:39.792+0700] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-09-01T07:08:39.814+0700] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): modify_data> on 2024-08-31 00:00:00+00:00
[2024-09-01T07:08:39.820+0700] {standard_task_runner.py:64} INFO - Started process 224862 to run task
[2024-09-01T07:08:39.823+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'scrap_mobil123', 'modify_data', 'scheduled__2024-08-31T00:00:00+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/mobil123/scrap_mobil123.py', '--cfg-path', '/tmp/tmp_1ct8ae4']
[2024-09-01T07:08:39.825+0700] {standard_task_runner.py:91} INFO - Job 214: Subtask modify_data
[2024-09-01T07:08:39.906+0700] {task_command.py:426} INFO - Running <TaskInstance: scrap_mobil123.modify_data scheduled__2024-08-31T00:00:00+00:00 [running]> on host BrightHorizons.
[2024-09-01T07:08:40.058+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrap_mobil123' AIRFLOW_CTX_TASK_ID='modify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-31T00:00:00+00:00'
[2024-09-01T07:08:40.059+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-01T07:08:40.091+0700] {base.py:84} INFO - Using connection ID 'spark_local' for task execution.
[2024-09-01T07:08:40.093+0700] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master local[*] --name arrow-spark --deploy-mode client /home/miracle/mobil/spark/mobil123.py
[2024-09-01T07:08:42.466+0700] {spark_submit.py:579} INFO - 24/09/01 07:08:42 WARN Utils: Your hostname, BrightHorizons resolves to a loopback address: 127.0.1.1; using 10.255.255.254 instead (on interface lo)
[2024-09-01T07:08:42.471+0700] {spark_submit.py:579} INFO - 24/09/01 07:08:42 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-09-01T07:08:43.548+0700] {spark_submit.py:579} INFO - python3: can't open file '/home/miracle/mobil/spark/mobil123.py': [Errno 2] No such file or directory
[2024-09-01T07:08:43.570+0700] {spark_submit.py:579} INFO - 24/09/01 07:08:43 INFO ShutdownHookManager: Shutdown hook called
[2024-09-01T07:08:43.572+0700] {spark_submit.py:579} INFO - 24/09/01 07:08:43 INFO ShutdownHookManager: Deleting directory /tmp/spark-eaf26fc3-1570-4991-8872-f691454348f0
[2024-09-01T07:08:43.607+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-01T07:08:43.626+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --name arrow-spark --deploy-mode client /home/miracle/mobil/spark/mobil123.py. Error code is: 2.
[2024-09-01T07:08:43.633+0700] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=scrap_mobil123, task_id=modify_data, run_id=scheduled__2024-08-31T00:00:00+00:00, execution_date=20240831T000000, start_date=20240901T000839, end_date=20240901T000843
[2024-09-01T07:08:43.652+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 214 for task modify_data (Cannot execute: spark-submit --master local[*] --name arrow-spark --deploy-mode client /home/miracle/mobil/spark/mobil123.py. Error code is: 2.; 224862)
[2024-09-01T07:08:43.695+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-09-01T07:08:43.725+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-01T07:08:43.732+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
